ymer/src/main/java/com/avanza/ymer/PerformedOperationMetrics.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2015 Avanza Bank AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.avanza.ymer;
+
+import java.util.concurrent.atomic.AtomicLong;
+
+public class PerformedOperationMetrics implements PerformedOperationMetricsMBean, PerformedOperationsListener {
+
+	private AtomicLong numInserts = new AtomicLong();
```

Comment:
Maybe using `LongAdder` would be a better fit?

Reply:
I've tried reading up on `LongAdder` and as far as I can see, the benefits are in usages where several threads updates the values in parallel. I can change to `LongAdder`, but as far as I know, we only use one thread handling the redo logs so I think the gain wouldn't be that huge

Reply:
Changed to `LongAdder` now

Reply:
Ah, you're right, currently we are only using one thread so there probably isn't any real gain.